src/components/ClearAllTags.tsx

```diff
@@ -4,10 +4,11 @@ const ClearAllTags = (props: {
   classNames: {
     clearAll: string,
   },
+  ariaLabel?: string,
```

Comment:
In `React` all ARIA attribute names are the same as in HTML hence it is better we follow the same convention here as the DOM instead of camel case

Reply:
done https://github.com/react-tags/react-tags/blob/8471fa6e74bc50938dc091d1cf1f484eaa17a200/src/components/ClearAllTags.tsx#L7